frontend/src/app/trees/forests/forest-finder/forest-finder.component.spec.ts

```diff
@@ -1,59 +1,58 @@
 import { async, ComponentFixture, TestBed, inject } from '@angular/core/testing';
-import { Ng2AutoCompleteModule} from 'ng2-auto-complete';
+import { Ng2AutoCompleteModule } from 'ng2-auto-complete';
 import { ForestFinderComponent } from './forest-finder.component';
 import { RouterTestingModule } from '@angular/router/testing';
-import { NO_ERRORS_SCHEMA} from '@angular/core';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { ForestService } from '../../_services/forest.service';
 import { RemovePuncPipe } from './remove-punc.pipe';
-import {HttpModule, XHRBackend, ResponseOptions, Response} from '@angular/http';
-import {MockBackend} from '@angular/http/testing';
-import {ScalarObservable} from 'rxjs/observable/ScalarObservable';
-import {SpacesToDashesPipe} from '../../../_pipes/spaces-to-dashes.pipe';
-import { MockActivatedRoute, MockRouter} from '../../../_mocks/routes.mock';
+import { HttpModule, XHRBackend, ResponseOptions, Response } from '@angular/http';
+import { MockBackend } from '@angular/http/testing';
+import { ScalarObservable } from 'rxjs/observable/ScalarObservable';
+import { SpacesToDashesPipe } from '../../../_pipes/spaces-to-dashes.pipe';
+import { MockActivatedRoute, MockRouter } from '../../../_mocks/routes.mock';
+import { MockService } from '../../../_services/mock.service';
+import { UtilService } from '../../../_services/util.service';
 
 describe('ForestFinderComponent', () => {
   let component: ForestFinderComponent;
   let fixture: ComponentFixture<ForestFinderComponent>;
   let mockActivatedRoute: MockActivatedRoute;
   let mockRouter: MockRouter;
+  let mockService: MockService;
 
   const mockResponse = [
     {
-      'id': 1,
-      'forestName': 'Forest National Forest',
-      'description': 'ForestName'
+      id: 1,
+      forestName: 'Forest National Forest',
+      description: 'ForestName'
     }
   ];
 
-  beforeEach(async(() => {
-    mockActivatedRoute = new MockActivatedRoute({data: {}});
-    mockRouter = new MockRouter();
-    TestBed.configureTestingModule({
-      declarations: [ ForestFinderComponent, RemovePuncPipe, SpacesToDashesPipe],
-      imports: [ HttpModule, Ng2AutoCompleteModule, RouterTestingModule ],
-      providers: [
-        { provide: ForestService, useClass: ForestService },
-        { provide: XHRBackend, useClass: MockBackend }
-      ],
-      schemas: [ NO_ERRORS_SCHEMA]
+  beforeEach(
+    async(() => {
+      mockActivatedRoute = new MockActivatedRoute({ data: {} });
+      mockRouter = new MockRouter();
+      mockService = new MockService();
+      TestBed.configureTestingModule({
```

Comment:
can this leverage the testService that was created for some of the other tests?

Reply:
We created the test service prior to omitting .spec files from code climate. We actually think the code is more readable without leveraging the test service @lauraGgit